fixtures/nodejs-hybrid-app/tests/index.test.ts

```diff
@@ -48,4 +48,19 @@ describe("nodejs compat", () => {
 			await stop();
 		}
 	});
+
+	test("crypto.X509Certificate is implemented", async ({ expect }) => {
```

Comment:
NIT the other tests use `it()`; did you switch to `test()` for a specific reason?

Reply:
"test crypto.X509Certificate is implemented" sounds better that "it crypto.X509Certificate is implemented"
but no strong opinion, I can it it if you prefer.

Reply:
No worries. Just interested